osism/commands/manage.py

```diff
@@ -225,6 +225,12 @@ def get_parser(self, prog_name):
             help="Hide images that should be deleted",
             action="store_true",
         )
+        parser.add_argument(
+            "--delete",
+            default=False,
+            help="Delete images that should be deleted",
+            action="store_true",
```

Comment:
the image-manager seems to have an additional `--yes-i-really-know-what-i-do` option, are you sure we want to go with just the simple option in this scenario?

Reply:
I think it makes sense to always add `--yes-i-really-know-what-i-do` when `--delete` is set in the OSISM CLI.